tests/integrate_test/cita_tls_basic.sh

```diff
@@ -16,12 +16,7 @@ cd ${BINARY_DIR}
 echo "DONE"
 
 ################################################################################
-echo -n "1) cleanup   ...  "
```

Comment:
Why remove these cleanup?

Reply:
It's useless since every test is an independent environment. 

Why did we need it before? because all tests run on one environment.

the snapshot test does not always work when run after other tests, maybe the cleanup is not really `cleanup`

Reply:
OK